content/api/dashboard_lists/code_snippets/result.api-dashboard-list-get-all.rb

```diff
@@ -0,0 +1,35 @@
+[
+    "200",
+    {
+        "dashboard_lists" => [
+            {
+                "is_favorite" => true,
+                "name" => "My Dashboard List B",
+                "dashboard_count" => 22,
+                "author" => {
+                    "handle" => "author.name@datadoghq.com",
```

Comment:
Let's keep this consistent with the other api doc example code and use test1@datadoghq.com and test2@datadoghq.com